app/models/manageiq/providers/ovirt4/infra_manager.rb

```diff
@@ -0,0 +1,29 @@
+require 'ovirtsdk4'
+
+class ManageIQ::Providers::Ovirt4::InfraManager < ManageIQ::Providers::InfraManager
```

Comment:
Yeah, thought so, but I'm not sure if this is the right approach to this.

This way you introduce a new provider class that is only useful with an existing provider.

So, if you want to test the ovirt-4 sdk now, do it with a feature toggle.
For the strategy for introducing a brand new provider or moving/renaming the redhat one, we should discuss that first with @blomquisg and maybe @Fryguy and @chessbyte